fl4health/datasets/skin_cancer/load_data.py

```diff
@@ -0,0 +1,204 @@
+# The following code is adapted from the medical_federated GitHub repository by Seongjun Yang et al.
```

Comment:
For this script and the preprocess script below, do you mind adding in function documentation in the Google docstring format. There is a link describing the style in the CONTRIBUTION.MD file, but there are also a fair bit of examples throughout the library.

Reply:
Preprocess has been changed significantly, but I have added google docstring comments in both of the files